pkg/storaged/index.html

```diff
@@ -844,7 +857,7 @@ <h4 class="modal-title">{{Title}}</h4>
             {{#Action}}
             {{#Danger}}<div class="modal-footer-danger">{{.}}</div>{{/Danger}}
             <div class="spinner spinner-sm pull-left" hidden></div>
-            <button class="btn btn-default" translatable="yes" data-dismiss="modal">Cancel</button>
+            <button class="btn btn-default" translatable="yes" data-action="cancel">Cancel</button>
```

Comment:
This change is seemingly unrelated, and could at least do with an explanation in the commit log? Or even a separate commit? 5 lines down continues to use `data-dismiss`, and so does the error-popup.

Reply:
Good catch!  5 lines down shoudl also use "data-action".  Now the dialog code needs to do some clean up when it is closed, so we need to put a real handler on the Cancel and Close buttons.

Reply:
Fixed.